app/i18n/en/home.json

```
@@ -1,15 +1,54 @@
 {
-  "titleCF": "CaptainFact - Collaborative Fact Checking platform",
-  "presentationTitle": "Let's check the Internet",
-  "presentation": "\n    The Internet is filled with rich and abundant data. Info travels fast and we don't\n    have the time to check that everything we see and hear is correct.\n    <2></2>\n    <4></4>\n    <0>CaptainFact</0> is all about people debating and helping each other to\n    verify that data by sourcing and voting on it, and chatting about facts.  \n  ",
+  "titleCF": "Collaborative Fact Checking platform",
+  "presentationTitle": "To train a critical mind, improve the quality of information and decision-making.<1 />Against fake news, fraud and disinformation",
+  "presentation": "An online platform that is open source, transparent and open to different perspectives.  It supports active citizenship and respects your privacy.",
+  "presentationTextButton1": "CaptainFact alerts you when you consult false or contested information on video.",
+  "presentationTextButton2": "CaptainFact is participative: sign up to help verify the information.",
+  "headerPunchline": "Against fake news, fraud and disinformation",
+  "headerPunchline2": "To train a critical mind, improve the quality of information and decision-making.",
+  "helpCaptain1": "Help CaptainFact fight disinformation",
+  "helpCaptain2": "by improving or developing the platform or by spreading the word about our work.",
+  "example1Title1": "Viewers",
+  "example1Title2": "For vigilant YouTube’s spectators.",
```

```suggestion
  "example1Title2": "For vigilant spectators.",
```
(The genitive s was an error.)